flask_graphql_auth/decorators.py

```diff
@@ -79,6 +79,30 @@ def verify_refresh_jwt_in_argument(token):
     ctx_stack.top.jwt = jwt_data
 
 
+def _extract_token_value(request_params, request_headers, token_name="token"):
+    """
+    Extract token value from the request's params by specific name.
+
+    If token is not passed in request params then the value of
+    `Bearer token` from `Authorization` header is returned.
+
+    As a side effect, when token is found in request params
+    then it's removed from the dict.
+
+    :param request_params: Request params as dict
+    :param request_headers: Request headers as dict
+    :param token_name: The name of the JWT token param
+    :return: Token value as a string (empty string if token is not found)
+    """
+    if token_name in request_params:
+        return request_params.pop(token_name)
+    else:
+        authorization_header = request_headers.get("Authorization", "")
+        if authorization_header.lower().startswith("bearer"):
```

Comment:
Please use app configuration to get token location and token prefix, not hard-coded value